src/postgraphql/http/__tests__/createPostGraphQLHttpRequestHandler-test.js

```diff
@@ -1,8 +1,5 @@
-jest.mock('../setupRequestPgClientTransaction')
```

Comment:
It looks like this is why the tests were failing 😊

Before we were mocking the actual setup behavior so the function would not actually run. I’m fine with this change 👍